R/plot-histogram.R

```diff
@@ -0,0 +1,66 @@
+set.seed(1234)
```

Comment:
Code that does not belong in the package should not be under R otherwise it will be exported with the package

Reply:
Yes this has been replaced